tests/ts/lsfd/mkfds-raw

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
The test script needs to be executable

Reply:
Note: the existing `mkfds-netns` is also not being tested at the moment and fails when enabled.

Reply:
Thank you for reviewing.
What do I have to is "chmod a+x ..."?

Reply:
> What do I have to is "chmod a+x ..."?

Yes

Reply:
FYI I get the following failure with `lsfd/mkfds-netns` on kernel 6.1.9, it seems to work on CI though:
```diff
--- /home/t-8ch/src/util-linux/tests/expected/lsfd/mkfds-netns	2022-12-07 20:49:19.989976710 +0000
+++ /home/t-8ch/src/util-linux/build-meson/tests/output/lsfd/mkfds-netns	2023-02-03 16:46:53.868615448 +0000
@@ -1,4 +1,4 @@
-    net
-net:[4026531840] == net:[4026531840]: 0
-net:[4026531840] == net:[4026531840]: 0
-net:[4026531840] == net:[4026531840]: 0
+
+ == : 0
+ == net:[4026531840]: 1
+ == net:[4026531840]: 1
```
Is this something obvious you see? Otherwise I'll investigate.

Reply:
Turns out I don't have nsfs mounted. And I don't even have it in /proc/filesystems.

Reply:
So nsfs is only mounted (automatically) in certain cases: https://unix.stackexchange.com/a/559723
But the logic in `stat2class` expects the `nsfs` filesystem to always be mounted.

Reply:
@t-8ch It seems that we must merge #2056 before this one.

Reply:
If you drop the `chmod +x tests/ts/lsfd/mkfds-netns` from this PR they should be independent.

Reply:
Dropped the commit. Thank you.